src/commands/clan/skills.ts

```diff
@@ -12,169 +12,185 @@ import {
   getRaidLog,
 } from "./_dread";
 
-const KILLMATCHER = /([A-Za-z0-9\-\_ ]+)\s+\(#\d+\)\s+defeated\D+(\d+)/;
-const SKILLMATCHER = /([A-Za-z0-9\-\_ ]+)\s+\(#\d+\)\s+used the machine/;
+const SKILL_KILL_MATCHER = /([A-Za-z0-9\-\_ ]+)\s+\(#\d+\)\s+(defeated\D+(\d+)|used the machine)/;
 
 export const data = new SlashCommandBuilder()
   .setName("skills")
   .setDescription("Get a list of everyone currently elgible for Dreadsylvania skills.");
 
-async function parseCurrentLogs(mapToUpdate: Map<string, PlayerData>) {
-  for (let clan of DREAD_CLANS) {
-    const raidLog = await getRaidLog(clan.id);
-    if (!raidLog) throw "Clan inaccessible";
```

Comment:
you seem to have swapped out this starting line error for an inaccessible clan with a catch at the very end. given how much time raidlog parsing takes, i'd prefer something upfront; probably just put the throw inside `getParticipationFromCurrentRaid`?

Reply:
I'll throw inside getRaidLog actually. Good point though

Reply:
This error was never actually being surfaced though, it is worth noting